Signal/translations/en.lproj/Localizable.strings

```diff
@@ -783,6 +783,9 @@
 /* No comment provided by engineer. */
 "NETWORK_ERROR_RECOVERY" = "Please check you're online and try again.";
 
+/* Indicates to the user that censorship circumvention has been activated. */
+"NETWORK_STATUS_CENSORSHIP_CIRCUMVENTION_ACTIVE" = "Censorship Circumvention: On";
```

Comment:
I'm not wild about the copy but as you can see in the screenshot it was hard to find copy that would fit.